server/server.go

```diff
@@ -272,8 +275,15 @@ func configureStorage(logger *logrus.Logger, conf *Config) (storage Storage, err
 	if conf.RedisURL != "" {
 		return NewRedisStorage(conf.RedisURL)
 	}
+	if conf.SqliteURL != "" {
+		return NewSqliteStorage(conf.SqliteURL)
+	}
+	if conf.Snapshot {
+		_ = os.Mkdir(conf.DBPath, os.ModePerm)
+		return NewSqliteStorage(fmt.Sprintf("%s/snapshot.sqlite", conf.DBPath))
```

Comment:
Couldn't we use snapshots with in-memory as well? Sqlite allows you having multiple in-memory instances, we could use `file:snapshot1-db?mode=memory&cache=shared` I think that could work, see here https://www.sqlite.org/inmemorydb.html

Reply:
hmm, I didn't know about that, let me check if it works, can be very good idea to implement.